src/certs/ca/cert/v1.rs

```diff
@@ -2,6 +2,13 @@
 
 use super::*;
 
+use serde::{Deserialize, Serialize};
+
+serde_big_array::big_array! {
+    BigArray;
+    256, 512,
```

Comment:
I see this gets moved down in the file in a future commit; why isn't it placed there initially?

Reply:
I think I ended up moving it near the macro invocation because it is only used for the expanded contents of the macro and I wanted to keep code generation mechanisms close together for easy editing. Easier to "see" where things come from.